<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-B07LBES" failures="1" tests="1" name="Registered Vehicle Test" time="0.841" errors="0" timestamp="2025-02-24T17:37:13 IST">
  <testcase classname="com.twowheeler.testscenario.New_Business" name="NewVehicle" time="0.0">
    <failure type="java.lang.RuntimeException" message="java.lang.NegativeArraySizeException: -1">
      <![CDATA[java.lang.RuntimeException: java.lang.NegativeArraySizeException: -1
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.NegativeArraySizeException: -1
at com.twowheeler.utilities.TestUtil.getData(TestUtil.java:48)
... 15 more
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- NewVehicle -->
</testsuite> <!-- Registered Vehicle Test -->
